editor/src/components/editor/import-wizard/components.tsx

```diff
@@ -82,7 +80,8 @@ function OperationChildrenList({ operation }: { operation: ImportOperation }) {
           successFn={dependenciesSuccessFn}
           successTextFn={dependenciesSuccessTextFn}
         />
-      ) : operation.type === 'checkRequirements' ? (
+      ) : operation.type === 'checkRequirementsPreParse' ||
+        operation.type === 'checkRequirementsPostParse' ? (
```

Comment:
I think this would be better served with a function that returns these components that has an exhaustive switch, rather than chaining ternaries. Then new operation types get handled appropriately and I think it would make this expression a little bit less complicated.